base/src/test/java/com/mindera/skeletoid/logs/LOGUnitTest.kt

```diff
@@ -134,16 +134,18 @@ class LOGUnitTest {
         appenders.add(appenderB)
         appenders.add(appenderC)
         init(context, packageName, appenders)
-        d(TAG, TEXT)
+        d(this, TAG, TEXT)
 
         //This is ugly.. but I don't see another way.
-        val log = String.format(
-            LOG_FORMAT_4ARGS,
+        val headers = listOfNotNull(
             TAG,
-            getObjectHash(TAG),
+            getObjectHash(this),
             ThreadUtils.currentThreadName,
-            getLogString(TEXT)
-        )
+        ).joinToString(separator = " ")
+
+        val log = String.format(LoggerManager.LOG_FORMAT, headers, getLogString(
+            TEXT
+        ))
```

Comment:
We are repeating this code in a lot of places. Maybe we could create a small function.

```kotlin
private fun headers() = 
    listOfNotNull(TAG, getObjectHash(this), ThreadUtils.currentThreadName)
        .joinToString(separator = " ")

private fun stringLog() = String.format(LOG_FORMAT, headers(), getLogString(TEXT))
```

Also, the call to `getLogString` seems unnecessary. Can't we just pass `TEXT`?